apps/studio/src/components/CmsSidebar/CmsSidebarContainer.tsx

```diff
@@ -0,0 +1,47 @@
+import type { ReactElement, ReactNode } from "react"
+import { Box, Grid, GridItem } from "@chakra-ui/react"
+
+export interface CmsSidebarContainerProps {
+  children: ReactNode
+  sidebar: ReactElement
+}
+
+export function CmsSidebarContainer({
+  children,
+  sidebar,
+}: CmsSidebarContainerProps) {
+  return (
+    <Grid
+      templateAreas={`'sidebar main'`}
+      templateColumns="auto 1fr"
+      width="100%"
+    >
```

Comment:
why the choice of `Grid` here over something else? this defines a 2 col layout which i thought might also work using something like a `HStack`